src/Symfony/Component/Clock/MockClock.php

```diff
@@ -47,6 +47,21 @@ public function sleep(float|int $seconds): void
         $this->now = (new \DateTimeImmutable($now, $timezone))->setTimezone($timezone);
     }
 
+    public function modify(string $now): void
+    {
+        try {
+            $modifiedNow = $this->now->modify($now);
+        } catch (\Throwable) {
+            throw new \InvalidArgumentException(sprintf('Invalid format: "%s". Could not modify MockClock.', $now));
+        }
+
+        if (false === $modifiedNow) {
```

Comment:
The try/catch above assumes a throwing error handler, better not. Better not also catch all throwables.
The correct code looks this to me:
`if (false === $modifiedNow = @$this->now->modify($now)) {`